src/PropertyHandler.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Livewire;
+
+abstract class PropertyHandler
```

Comment:
Could this just be an interface instead if it only contains abstract methods?

Reply:
Could this just be an interface instead if we only contains abstract methods?

Reply:
@jonassiewertsen I'm not sure if Caleb will agree to an interface/abstract class.

I know we've discussed it before, but I think changing over to `method_exists` checks to see if the method exists on a resolver would be better (Caleb does that in his experiment).

Then people can implement as many or little methods as it required for their property type.

I'm ok if you don't change this yet though, until we're ready for Caleb to provide feedback and he can decide which he would prefer.